<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GraphSetupDialog</class>
    <widget class="QDialog" name="GraphSetupDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>550</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Graph Setup</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="graphOptionsBox">
                    <property name="title">
                        <string>Graph Options</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QCheckBox" name="directedCheckBox">
                                <property name="text">
                                    <string>Directed</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="weightedCheckBox">
                                <property name="text">
                                    <string>Weighted</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="vertexCountLabel">
                                <property name="text">
                                    <string>Number of vertices:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSpinBox" name="vertexCountSpinBox">
                                <property name="minimum">
                                    <number>1</number>
                                </property>
                                <property name="maximum">
                                    <number>1000</number>
                                </property>
                                <property name="value">
                                    <number>5</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="edgesLabel">
                                <property name="text">
                                    <string>Edges (format: src dst weight):</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPlainTextEdit" name="edgesTextEdit"/>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>